tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AspNet/AspNetMvc5Tests.cs

```diff
@@ -111,18 +109,35 @@ public AspNetMvc5TestsModuleOnlyVirtualAppIntegratedWithFeatureFlag(IisFixture i
         }
     }
 
+    [Collection("IisTests")]
+    public class AspNetMvc5Tests128BitTraceIds : AspNetMvc5Tests
+    {
+        public AspNetMvc5Tests128BitTraceIds(IisFixture iisFixture, ITestOutputHelper output)
```

Comment:
I'm confused...is the snapshot the same before and after setting the `TraceId128BitGenerationEnabled` flag? Is that expected?

Reply:
Nope, you are correct! This are the tests I'm trying to finish (wip), but having other issues right now. Once they fail "correctly" I can grab the updated snapshots.

Reply:
Done!